libs/cart/testing/src/drivers/in-memory/cart-shipping-information/cart-shipping-information.service.ts

```diff
@@ -0,0 +1,30 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs';
+
+import {
+  DaffCart,
+  DaffCartShippingInformationServiceInterface,
+  DaffCartShippingRate
+} from '@daffodil/cart';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class DaffInMemoryCartShippingInformationService implements DaffCartShippingInformationServiceInterface<DaffCartShippingRate, DaffCart> {
+  url = '/api/cart-shipping-information';
+
+  constructor(private http: HttpClient) {}
+
+  get(cartId: DaffCart['id']): Observable<DaffCartShippingRate> {
+    return this.http.get<DaffCartShippingRate>(`${this.url}/${cartId}`);
+  }
+
+  update(cartId: DaffCart['id'], info: Partial<DaffCartShippingRate>): Observable<DaffCart> {
```

Comment:
This should return an `Observable<Partial<DaffCart>>`

Reply:
Changed